typings/index.d.ts

```diff
@@ -918,9 +918,15 @@ declare module 'discord.js' {
 		public remove(user?: UserResolvable): Promise<MessageReaction>;
 	}
 
-	export class NewsChannel extends TextChannel {
+	export class NewsChannel extends TextBasedChannel(GuildChannel) {
```

Comment:
This is missing some properties that went into the mixin in master. (Compare with definition of TextChannel)

I personally would prefer this still extending `TextChannel` as we can't drop the rateLimitPerUser properties anyway.

Reply:
RIP, I'll roll it back

Reply:
Resolved in eb87a19